sdk/communication/pom.xml

```diff
@@ -42,6 +43,11 @@
           <artifactId>azure-communication-identity</artifactId>
           <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-identity;current} -->
         </dependency>
+        <dependency>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-communication-networktraversal</artifactId>
+          <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-networktraversal;current} -->
```

Comment:
1.0.0-beta.1? Please also check that the versioning for this new package is consistent everywhere